src/umpire/strategy/mixins/Inspector.cpp

```diff
@@ -29,7 +29,7 @@ Inspector::registerAllocation(
   m_allocation_count++;
 
   if (m_current_size > m_high_watermark) {
-    m_high_watermark = m_current_size;
+    m_high_watermark.store(m_current_size, std::memory_order_relaxed);
```

Comment:
I don't think we want to make these atomics - it's an unnecessary performance hit (but I understand it's required to shut up the analyzer). Is there an alternative?

Reply:
Well we should talk about it this afternoon, because if it's a data race, then I should either fix it (and what that might be) or, if it's a known data race like the 'umpire_ver_5_found', then I can make the sanitizer ignore it explicitly.